proxy.go

```diff
@@ -239,3 +251,24 @@ func copyResponseHeaders(w http.ResponseWriter, resp *http.Response) {
 	w.Header().Del("Transfer-Encoding")
 	w.Header().Del("Upgrade")
 }
+
+type dialError struct {
+	network, address string
+	err              error
+}
+
+func (e *dialError) Error() string {
+	return fmt.Sprintf("error while dialing %q %q: %s", e.network, e.address, e.err.Error())
+}
+
+func (e *dialError) Unwrap() error {
+	return e.err
+}
+
+func dialContext(ignored context.Context, network, address string) (net.Conn, error) {
```

Comment:
"ignored" is typically spelt "_" in Go

Reply:
Done

Reply:
I started wondering why you didn't use the context parameter, and use `net.DialContext()` instead. So I looked where it was called from so I could understand what the context arg would be, and I could not find one. This function is unused.

BTW, golangci-lint also tells you this ;-)

Reply:
Oops, looks like I forgot to clean that one up. We should add golangci-lint to GitHub Actions, and I've created #64 to track that.

Reply:
Done (removed the unused `dialContext` function)